src/Core_Command.php

```diff
@@ -642,6 +642,9 @@ private function _multisite_convert( $assoc_args ) {
 			$assoc_args['subdomains']
 		);
 
+		// Fetch what the actual site id turned out to be.
+		$site_id = get_current_site()->id;
```

Comment:
The multisite-specific code has not been loaded at this point, so the function `get_current_site()` does not exist.

See test failures here: https://travis-ci.org/wp-cli/core-command/jobs/524578309#L668

You might be able to directly reference the `$current_site` global, but you'll have to check first whether that is populated.

Reply:
This is resolved in the latest code. I found that it was necessary to use a direct DB query instead, in order to get the correct site id.